graylog2-server/src/main/java/org/graylog2/bindings/GraylogNodeModule.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.bindings;
+
+import com.google.common.eventbus.EventBus;
+import com.google.inject.Scopes;
+import org.glassfish.jersey.internal.guava.Sets;
+import org.graylog2.GraylogNodeConfiguration;
+import org.graylog2.audit.AuditBindings;
+import org.graylog2.configuration.EventBusConfiguration;
+import org.graylog2.configuration.MongoDbConfiguration;
+import org.graylog2.jackson.InputConfigurationBeanDeserializerModifier;
+import org.graylog2.plugin.LocalMetricRegistry;
+import org.graylog2.plugin.inject.Graylog2Module;
+import org.graylog2.plugin.system.FilePersistedNodeIdProvider;
+import org.graylog2.plugin.system.NodeId;
+import org.graylog2.plugin.system.SimpleNodeId;
+import org.graylog2.security.encryption.EncryptedValueService;
+import org.graylog2.shared.bindings.ObjectMapperModule;
+import org.graylog2.shared.bindings.SchedulerBindings;
+import org.graylog2.shared.bindings.ServerStatusBindings;
+import org.graylog2.shared.bindings.providers.EventBusProvider;
+
+import java.util.Set;
+
+/**
+ * <p>Guice module that contains all necessary bindings to start a basic node in a Graylog cluster.</p>
+ *
+ * <ul>
```

Comment:
Let's either update the list of bindings to match the module or remove the list.

Reply:
Ping.

Reply:
i removed the list here and added documentation to the config toggles in `GraylogNodeConfiguration`

Reply:
Thanks!